clients/docs/go.rst

```diff
@@ -0,0 +1,134 @@
+~~~~~~~~~~~~~
+
+.. include:: includes/client-example-prerequisites.rst
+
+
+Setup
+-----
+
+#. Clone the `confluentinc/examples GitHub repository
+   <https://github.com/confluentinc/examples>`__ and check out the
+   :litwithvars:`|release|-post` branch.
+
+   .. codewithvars:: bash
+
+      git clone https://github.com/confluentinc/examples
+      cd examples
+      git checkout |release|-post
+
+#. Change directory to the example for Go.
+
+   .. code-block:: bash
+
+      cd clients/cloud/go/
+
+#. .. include:: includes/client-example-create-file.rst
+
+
+Basic Producer and Consumer
+---------------------------
+
+In this example, the producer writes Kafka data to a topic in your Kafka
+cluster. Each record has a key representing a username (for example, ``alice``)
+and a value of a count, in JSON format (for example, ``{"Count": 0}``). The
+consumer reads the same topic and keeps a rolling sum of the counts as it
+processes each record.
+
+
+Produce Records
+~~~~~~~~~~~~~~~
+
+#. Run the producer, passing in arguments for:
+
+   - the local file with configuration parameters to connect to your Kafka cluster
+   - topic name
+
+   .. code-block:: bash
+
+      go build producer.go
+
+   You should see:
+
+   .. code-block:: bash
+
+      Preparing to produce record: alice   {"Count": 0}
+      Preparing to produce record: alice   {"Count": 1}
+      Preparing to produce record: alice   {"Count": 2}
+      Preparing to produce record: alice   {"Count": 3}
+      Preparing to produce record: alice   {"Count": 4}
+      Preparing to produce record: alice   {"Count": 5}
+      Preparing to produce record: alice   {"Count": 6}
+      Preparing to produce record: alice   {"Count": 7}
+      Preparing to produce record: alice   {"Count": 8}
+      Preparing to produce record: alice   {"Count": 9}
+      Successfully produced record to topic test1 partition [0] @ offset 0
+      Successfully produced record to topic test1 partition [0] @ offset 1
+      Successfully produced record to topic test1 partition [0] @ offset 2
+      Successfully produced record to topic test1 partition [0] @ offset 3
+      Successfully produced record to topic test1 partition [0] @ offset 4
+      Successfully produced record to topic test1 partition [0] @ offset 5
+      Successfully produced record to topic test1 partition [0] @ offset 6
+      Successfully produced record to topic test1 partition [0] @ offset 7
+      Successfully produced record to topic test1 partition [0] @ offset 8
+      Successfully produced record to topic test1 partition [0] @ offset 9
+      10 messages were produced to topic test1!
+
+
+#. View the :devx-examples:`producer code|clients/cloud/go/producer.go`.
+
+
+Consume Records
+~~~~~~~~~~~~~~~
+
+#. Run the consumer, passing in arguments for:
+
+   - the local file with configuration parameters to connect to your Kafka
+     cluster
+   - the same topic name used in step 1
+   
+   .. code-block:: bash
+
+       go build consumer.go
```

Comment:
@tikimims I'm sorry I told you the wrong information yesterday.  The line below is required, it should follow immediately after the build

```
./consumer -f $HOME/.confluent/librdkafka.config -t test1		
```